lib/src/base-styles/_nx-text-link.scss

```diff
@@ -37,8 +37,8 @@
     // was changed to 4px to get it visually close to
     // var(--nx-border-radius) or (6px).
     border-radius: 4px;
-    outline: 1px solid var(--nx-color-interactive-border-focus);
-    outline-offset: 1px;
+    outline: 2px solid var(--nx-color-interactive-border-focus);
+    outline-offset: 2px;
```

Comment:
Looking at Sketch, it seems as though the focus border is tighter around the text (and if you focus on the example of NxCode inside of a NxTextLink, it leaves a small white space between the code's background colour and the focus border)

Reply:
> Looking at Sketch, it seems as though the focus border is tighter around the text (and if you focus on the example of NxCode inside of a NxTextLink, it leaves a small white space between the code's background colour and the focus border). So we may not need the 2px of `outline-offset`

Yup! I removed it here: 0ee5deb101caa7d5c4dac776f57c1b10c1a8998e